.teamcity/src/main/kotlin/projects/CheckProject.kt

```diff
@@ -72,8 +72,8 @@ class CheckProject(
     subProjectsOrder = subProjects
 
     cleanupRule(
-        historyDays = 14,
-        artifactsDays = 7,
+        historyDays = 28,
+        artifactsDays = 28,
```

Comment:
🤔 That means 4 times the size we keep now. I think I'd start with 14 days and then look at the usage.